code-challenges.test.js

```diff
@@ -15,24 +15,82 @@
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+describe("shortShuffle", () => {
+  const colors1 = ["purple", "blue", "green", "yellow", "pink"];
+  // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+  const colors2 = [
+    "chartreuse",
+    "indigo",
+    "periwinkle",
+    "ochre",
+    "aquamarine",
+    "saffron",
+  ];
+  // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+  it("Removes the first item from the array", () => {
+    expect(shortShuffle(colors1)).toEqual(
+      expect.not.arrayContaining(["purple"])
+    );
+    expect(shortShuffle(colors2)).toEqual(
+      expect.not.arrayContaining(["chartreuse"])
+    );
+  });
+  it("Shuffles the remaining items", () => {
+    expect(shortShuffle(colors1)).not.toEqual([
+      "blue",
+      "green",
+      "yellow",
+      "pink",
+    ]);
+    expect(shortShuffle(colors2)).not.toEqual([
+      "indigo",
+      "periwinkle",
+      "ochre",
+      "aquamarine",
+      "saffron",
+    ]);
+  });
+});
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// Declare a function called shortShuffle
+// Input: An array
+// Input: An array
+// Use .shift() to remove the first value at the 0 index of the array
+// Use a Fisher-Yates shuffle to rearrange the rest of the values in the array
```

Comment:
This is some very basic pseudocode that does not really explain the intent of the code below.  Fisher-yates is an excellent approach to this problem, just would like to see a more clear breakdown of how you intend to approach this method in your code.